app.js

```diff
@@ -0,0 +1,207 @@
+const menuIcon = document.querySelector('.menu-icon');
+const closeIcon = document.querySelector('.mobile-menu-contaner .close-icon');
+const mobileMenuCont = document.querySelector('.mobile-menu-contaner');
+const mainbar = document.querySelector('.main-bar');
+const linksa = document.querySelector('.mobile-menu-contaner ul ');
+
+// const scrollb=document.querySelector('body');
+// body.style.overflow = "hidden";
```

Comment:
- [ ] Kindly remove commented-out code because it will make your software vulnerable to threat 🐛 🐞

Reply:
@rica213  thank you for you positive and kind feedbacks as per your comment i remove all the commented code from index.html, style.css, app.js and also i adjust the mobile pop window and i try to check you using to mobile size and i attach them here 
![Screen Shot 2023-06-28 at 2 58 47 PM](https://github.com/ethred/Portfolio/assets/49729832/b617c806-6ae2-44ff-b130-e76364c59cd7)
![Screen Shot 2023-06-28 at 3 00 00 PM](https://github.com/ethred/Portfolio/assets/49729832/8c56c9f3-1c48-4445-9e0c-6b042adeff6f)